syntax = "proto2";

package lczero.training;

// Configuration for training algorithm and parameters.
message TrainingConfig {
  // Empty for now - will be populated with training-specific configuration
}